hyperon_das_atomdb/adapters/hash_table.py

```diff
@@ -619,3 +642,12 @@ def _calculate_composite_type_hash(self, composite_type: list) -> str:
                     _hash_copy
                 )
         return ExpressionHasher.composite_hash(composite_type)
+
+    def _remove_links_not_toplevel(self, matches: list) -> list:
```

Comment:
change the name to _filter_non_toplevel() because "filter" is more accurate than "remove" and there's no need to enforce the handles to be links.

Reply:
ok

Reply:
ok. got it